File: Byte&Buy.Desktop/src/ByteBuy.Services/DTO/Address/ShippingAddressResponse.cs
namespace ByteBuy.Services.DTO.Address;

public record ShippingAddressResponse(
    Guid Id,
    Guid CountryId,
    string Label,
    string Street,
    string HouseNumber,
    string PostalCity,
    string PostalCode,
    string City,
    string? FlatNumber,
    bool IsDefault
);

File: Byte&Buy.AngularSPA/src/app/core/dto/shipping-address/shipping-address-response.ts
import { Guid } from "guid-typescript"

export interface ShippingAddressResponse{
    id: Guid;
    countryId: Guid;
    label: string;
    street: string;
    houseNumber :string;
    postalCity: string;
    postalCode :string;
    city :string;
    country :string;
    flatNumber :string | null;
    isDefault: boolean;
}